src/tests/ftest/cart/tests_common.h

```diff
@@ -38,291 +38,41 @@ struct test_options {
 	int		delay_shutdown_sec;
 };
 
-static struct test_options opts = { .is_initialized = false };
+extern struct test_options opts;
```

Comment:
not a big fan of externs; i would prefer if possible to change it so that we have tc_get_opts() tc_set_opts() instead of accessing opts directly

Reply:
Created function tc_get_opts.  Do not need tcv_set_opts.  Functions changed where needed.